tests/testthat/test-envelope.R

```diff
@@ -2,3 +2,35 @@ test_that("envelope print() output", {
   msg <- envelope() %>% subject("Test message")
   expect_output(print(msg), "Date:         .*\nSubject:      Test message")
 })
+
+test_that("class envelope", {
+  object <- envelope()
+  expect_output(print(class(object)), "envelope")
```

Comment:
Would it not make more sense here to just use `expect_equal()` and don't use `print()`.

Reply:
Yes, that makes much more sense.  Thanks!  Done.